js/public_price_set_form.js

```diff
@@ -186,7 +186,7 @@ cj(function() {
     // So if  we still don't have a value for monetarySymbol, try getting
     // the first non-space string in the pricevalue div (this is thought to be
     // more likely to work in cases of multi-character symbols (e.g. "Lek")
-    CRM.percentagepricesetfield.monetarySymbol = cj('#pricevalue').html().split(' ')[0];
+    CRM.percentagepricesetfield.monetarySymbol = cj('#pricevalue').html().slice(0,1);
```

Comment:
@yashodha Yes, I see what you mean about the missing space character.
With this solution, and with what's happening on line 77, your total '$123.00' would appear as '$ 123.00', i.e. different from the format you've configured. That's probably not a big problem, but I think we could avoid it -- so that your total appears in the format you've configured -- by using this on line 189:
`CRM.percentagepricesetfield.monetarySymbol = cj('#pricevalue').html().replace(/[0-9,.]/g, '');`
and then removing the `+ ' ' +` on line 77.

See what I mean? Wouldn't you like that a little better?

Reply:
Well, correcting myself here. Spaces are also allowed as a thousands separator, so my regex isn't quite right. Let me think on this for a minute.